src/components/ClusterManager.js

```diff
@@ -451,18 +341,123 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
       div({ style: styles.row }, [
         div({ style: styles.label }, [
           `Cost: ${Utils.formatUSD(machineConfigCost(this.getMachineConfig()))} per hour`
-        ]),
-        div({ style: { marginLeft: 'auto' } }, [
-          busy && icon('loadingSpinner')
-        ]),
-        buttonSecondary({
-          style: { marginLeft: '1rem', marginRight: '1rem' },
-          onClick: () => this.toggleDropdown(false)
-        }, 'Cancel'),
-        buttonPrimary({ disabled: busy || !changed, onClick: () => this.createCluster() }, currentCluster ? 'Update' : 'Create')
-      ])
+        ])
+      ]),
+      busy && spinnerOverlay
```

Comment:
The spinnerOverlay prevents you from interacting with the page at all while the initial spinning up occurs (~5-6s). We used to have an spinning icon, and users could still interact with the app after clicking "update". Is there a way we can convey that the creation has started, but not have a delay?

Reply:
Good call, it now closes immediately and performs the create call in the background. Any error will still result in a popup message.